tests/unit/clients/dbt_runner/test_retry_logic.py

```
@@ -201,3 +201,114 @@ def test_successful_command_not_retried(self, mock_subprocess_run):
 
         assert mock_subprocess_run.call_count == 1
         assert result is True
+
+
+def _make_api_runner(**kwargs):
+    """Create an APIDbtRunner with deps/packages stubbed out."""
+    defaults = dict(
+        project_dir="/tmp/fake_project",
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/fake_project"

(S108)